tests/test_raft_paper.rs

```diff
@@ -815,7 +815,7 @@ fn test_vote_request() {
         r.step(m).expect("");
         r.read_messages();
 
-        for _ in 0..r.get_election_timeout() * 2 {
+        for _ in 1..r.get_election_timeout() * 2 {
```

Comment:
@siddontang @BusyJay Actually can you paste the failure result here? Does the previous test code triggers two elections? (previous only > election timeout can trigger an election. with the new code change, ==election might trigger an election. so range from 0 to 2*e might trigger two elections instead of one) I want to know for sure this is the actual issue.